snp_application/src/snp_widget.cpp

```diff
@@ -110,7 +110,15 @@ SNPWidget::SNPWidget(rclcpp::Node::SharedPtr rviz_node, QWidget* parent)
   bt_node_->declare_parameter<std::string>(BT_PARAM, "");
   bt_node_->declare_parameter<int>(BT_TIMEOUT_PARAM, 6000);  // seconds
   bt_node_->declare_parameter<std::string>(FOLLOW_JOINT_TRAJECTORY_ACTION, "follow_joint_trajectory");
+  bt_node_->declare_parameter<std::string>(MESH_FILE_PARAM, "");
+  bt_node_->declare_parameter<std::string>(REF_FRAME_PARAM, "");
```

Comment:
I don't think these are necessary because they are declared as a part of loading the plugins that use them